Assets/MixedRealityToolkit/_Core/Devices/Simulator/SimulatedController.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices;
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Devices.Simulator
+{
+    public struct SimulatedController : IMixedRealityController
```

Comment:
Could we also add a simulated Hand controller as well?

Reply:
Done